qiskit/assembler/assemble_schedules.py

```diff
@@ -203,6 +203,11 @@ def _assemble_instructions(
                 name=name)
             user_pulselib[name] = instruction.pulse.samples
 
+        # ignore delay instrs on acq channels (don't matter w/ absolute timing)
```

Comment:
This is a little misleading, since delays at the end of a sub-program do have an effect. This might be one case where a more detailed explanation (describing what would happen if this stayed) in the comments is warranted

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/6000/commits/e24adf163f5804576f2d85c7c5bf8a895238775b.